cloudify_cli/tests/test_env.py

```diff
@@ -124,19 +124,19 @@ def _make_mock_profile(self, profile_name='10.10.1.10'):
 
     def test_delete_profile(self):
         profile_path = self._make_mock_profile()
-        env.delete_profile('10.10.1.10')
+        env.delete_profile('10.10.1.11')
         self.assertFalse(os.path.isdir(profile_path))
 
     def test_profile_exists(self):
         self.assertFalse(env.is_profile_exists('non-existing-profile'))
 
     def test_profile_does_not_exist(self):
         self._make_mock_profile()
-        self.assertTrue(env.is_profile_exists('10.10.1.10'))
+        self.assertTrue(env.is_profile_exists('10.10.1.11'))
 
     def test_assert_profile_exists(self):
         self._make_mock_profile()
-        env.assert_profile_exists('10.10.1.10')
+        env.assert_profile_exists('10.10.1.11')
```

Comment:
All these IPs should probably be example/docs IPs- both for good form, and so that if we screw something up we don't hit any legitimate IPs with a bunch of random traffic.